NineChronicles.Headless.Executable/Program.cs

```diff
@@ -66,87 +66,101 @@ static void ConfigureSentryOptions(SentryOptions o)
 
         [PrimaryCommand]
         public async Task Run(
-            [Option("app-protocol-version", new[] { 'V' }, Description = "App protocol version token")]
+            [Option("app-protocol-version", new[] { 'V' },
+                Description = "App protocol version token.")]
             string appProtocolVersionToken,
-            [Option('G')]
+            [Option('G',
+                Description = "Genesis block path of blockchain. Blockchain is recognized by its genesis block.")]
             string genesisBlockPath,
-            bool noMiner = false,
-            [Option('H')]
+            [Option('H',
+                Description = "Hostname of this node for another nodes to access." +
+                              "This is not listening host like 0.0.0.0")]
             string? host = null,
-            [Option('P')]
+            [Option('P',
+                Description = "Port of this node for another nodes to access.")]
             ushort? port = null,
             [Option("swarm-private-key",
                 Description = "The private key used for signing messages and to specify your node. " +
                               "If you leave this null, a randomly generated value will be used.")]
             string? swarmPrivateKeyString = null,
+            [Option("workers", Description = "Number of workers to use in Swarm")]
+            int workers = 5,
+            [Option(Description = "Disable block mining.")]
+            bool noMiner = false,
+            [Option("miner-count", Description = "The number of miner task(thread).")]
+            int minerCount = 1,
             [Option("miner-private-key",
                 Description = "The private key used for mining blocks. " +
                               "Must not be null if you want to turn on mining with libplanet-node.")]
             string? minerPrivateKeyString = null,
             [Option(Description = "The type of storage to store blockchain data. " +
-                                  "If not provided, \"LiteDB\" will be used as default. " + 
+                                  "If not provided, \"LiteDB\" will be used as default. " +
                                   "Available type: [\"rocksdb\", \"memory\"]")]
             string? storeType = null,
+            [Option(Description = "Path of storage. If not provided, default path will be used.")]
```

Comment:
I'm not sure where the default path is. 🤔  If you found, could you attach the link on this thread?

Reply:
I've digged this and I finally found path from [9c-launchner](https://github.com/planetarium/9c-launcher)
From here, 9c-launcher gets store-path and passes to headless. ([Reference](https://github.com/planetarium/9c-launcher/blob/4822db531999d80411b031d5fca79f81318f7d82/src/main/main.ts#L943-L948))
I think Headless or Libplanet itself cannot create or get default store path 
We should notice that the `--store-path` is required when `--store-type` is persistent type e.g. `rocksdb`
I'll add this to documentation with next commit.